app/javascript/react/components/Calendar/Month.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import moment from "moment";
+
+import * as S from "./Month.style";
+
+import { yellow } from "../../assets/styles/colors";
+
+const getFullWeeksOfMonth = (year: number, month: number) => {
+  let start = moment([year, month]).startOf("month").startOf("week");
+  let end = moment([year, month]).endOf("month").endOf("week");
+
+  let weeks = [];
+  while (start <= end) {
+    let week = [];
+    for (let i = 0; i < 7; i++) {
+      week.push(start.clone().add(i, "days").format("YYYY-MM-DD"));
+    }
+    weeks.push(week);
+    start.add(1, "week");
+  }
+  return weeks;
+};
+
+const getDayNumber = (date: string) => {
+  return moment(date).format("D");
+};
+
+const Month = () => {
+  const weeks = getFullWeeksOfMonth(2024, 2);
+  console.log(weeks);
+
+  return (
+    <>
+      <S.ThreeMonths>
+        <S.Month>
+          <S.MonthName>February</S.MonthName>
+          {weeks.map((week) => (
+            <S.Week>
+              {week.map((day) => (
+                <S.Day shouldColor={true} color={yellow} isCurrentMonth={true}>
+                  <S.DayNumber shouldDisplay={true}>
+                    {getDayNumber(day)}
+                  </S.DayNumber>
+                  <S.Value shouldDisplay={true}>100</S.Value>
+                </S.Day>
+              ))}
+            </S.Week>
+          ))}
+        </S.Month>
+        <S.Month>
+          <S.MonthName>March</S.MonthName>
```

Comment:
Having a month view, I think we can properly iterate over an array of months we would like to consider and pass the names, or maybe use `https://momentjs.com/docs/#/displaying/format/` to get name based on the number which could be more error prone?

Reply:
Yes, I didn't plan to leave it hard-coded, just left it for another PR :)